stable/prometheus-blackbox-exporter/templates/prometheusrule.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
     app.kubernetes.io/managed-by: {{ $.Release.Service }}
     helm.sh/chart: {{ include "prometheus-blackbox-exporter.chart" $ }}
     {{- with .Values.prometheusRule.additionalLabels }}
-    {{- toYaml . | indent 4 }}
+    {{- toYaml . | nindent 4 }}
```

Comment:
This is nitpicky, but actually, before, we removed the newline, and now you remove the newline and add it back. Could you remove the leading newline `{{-` and use `indent`?

Reply:
```suggestion
    {{ toYaml . | indent 4 }}
```

Reply:
Don't worry if it creates error, let's not waste time on this and keep what you've done. To prove that it works, could you create a test in the ci directory?

Reply:
This is my misunderstanding. `indent` causes error when multiple labels were applied.